cron_jobs/caravan_cron.php

```diff
@@ -22,30 +23,36 @@
     if ($player->staff_manager->isHeadAdmin()) {
         if (isset($_GET['run_script']) && $_GET['run_script'] == 'true') {
             if ($system->environment === System::ENVIRONMENT_PROD) {
+                $debug = false;
+                if (isset($_GET['debug'])) {
+                    $debug = $system->db->clean($_GET['debug']);
```

Comment:
We could call this `dry_run` to be a little more indicative of what it does, but this is ok 👍